content/docs/2_cookbook/7_extensions/0_content-file-cleanup/cookbook-recipe.txt

```diff
@@ -40,61 +40,61 @@ $kirby = new Kirby;
 $kirby->impersonate('kirby');
 
 // Define your collection
-// Don't use `$site->index()` for thousands of pages
-$collection = $kirby->site()->index();
+$models = $kirby->site()->index(true);
```

Comment:
Don't really understand the use of the `$models` variable name. As if I was going to loop through different model types like pages, users and files at the same time.

Reply:
Changed it.

I removed the paragraph about large index as `$kirby->models()` is a greater and yields each model. To my understanding, this should work also for larger sites.